theory/particle physics/neutrinos.md

```diff
@@ -7,3 +7,55 @@ nav_order: 5
 ---
 
 ## Neutrinos
+
+### What are Neutrinos?
+
+[Neutrinos](#neutrinos) are ultrasmall, electrically neutral elementary particles. By being electrically neutral, neutrinos join the family of fundamental particles called [leptons](../particle%20physics/lepton%20sources.html), which include other members like electrons and muons. Neutrinos form that group of particles present in space in abundance but very hard to detect, since they interact very rarely with matter.
+
+### Properties of Neutrinos
+
+[Neutrinos](#neutrinos) come in three flavors or types: neutrino electron, neutrino muon, and neutrino tau. Each is associated with a charged lepton-electron, muon, and tau. Neutrinos have very negligible mass, probably zero mass. They also travel at almost the speed of light.
+
+The mass of the neutrino, as related to its energy, E, momentum, p, and speed, v, is given by the expression below:
+
+$$E^2 = (pc)^2 + (mc^2)^2$$
+
+where,
+
+$$E$$ = energy of the neutrino
+
+$$p$$ = momentum of the neutrino
+
+$$c$$ = the velocity of light
+
+$$m$$ = mass of the neutrino
+
+### How Do We Detect Neutrinos?
+
+By their very nature, [neutrinos](#neutrinos) can be detected only indirectly, by observing the particles they produce when interacting with matter. Really big neutrino detectors are constructed deep underground  to protect them from background radiation from [cosmic rays](../particle%20physics/cosmic%20rays.html). When a neutrino interacts with an atomic nucleus in the detector, it produces a secondary particle that may be observable.
+
+![Neutrino](../../assets/images/theory/particle%20physics/lepton%20sources/neutrinos/neutrinos.jpg)
```

Comment:
Give credits wherever necessary

Reply:
Please move the image to `../../assets/images/theory/particle%20physics/neutrinos.jpg`

Also, the image quality is terrible and the text illegible. Please find an alternative image or remove this